source/CoordinateConversion/ProAppCoordConversionModule/ViewModels/ProTabBaseViewModel.cs

```diff
@@ -31,8 +31,8 @@ namespace ProAppCoordConversionModule.ViewModels
 {
     public class ProTabBaseViewModel : TabBaseViewModel
     {
-        // This name should correlate to the name specified in Config.esriaddinx - Tool id="Esri_ArcMapAddinCoordinateConversion_MapPointTool"
-        internal const string MapPointToolName = "Esri_ArcMapAddinCoordinateConversion_MapPointTool";
+        // This name should correlate to the name specified in Config.esriaddinx - Tool id="ProAppCoordConversionModule_CoordinateMapTool"
+        internal const string MapPointToolName = "ProAppCoordConversionModule_CoordinateMapTool";
```

Comment:
Does not affect the single add-in. The config.daml for single add-in does have "ProAppCoordConversionModule_CoordinateMapTool" ID [line 60](https://github.com/Esri/military-tools-desktop-addins/blob/dev/source/MilitaryToolsForArcGISPro/Config.daml#L60)